### ISSUE TITLE
Why do you set 'requires_grad=False' in the 'features' layer of iresnet?

### ISSUE THREAD

**Issue by ygtxr1997** [deepinsight/insightface#1806]
'/insightface/recognition/arcface_torch/backbones/iresnet.py':
line 98:         self.features = nn.BatchNorm1d(num_features, eps=1e-05)
line 99:         nn.init.constant_(self.features.weight, 1.0)
line 100:       self.features.weight.requires_grad = False
--------------------------------------------------------------------------------------------------------
What was your purpose of setting 'self.features.weight.requires_grad' as 'False'? To avoid some problem of optimizing? Or others?

**Comment by special-yang** [comment]
why？
